lnschema_core/_core.py

```diff
@@ -174,6 +179,11 @@ class DObject(SQLModel, table=True):  # type: ignore
         sa_relationship_kwargs=dict(secondary=DObjectFeatures.__table__),
     )
     """Link to feature sets."""
+    folders: List["DFolder"] = Relationship(
```

Comment:
Are we sure about this? 🤔

Reply:
Also, you don't need quotation marks, here! `DFolder` is already defined, and hence, you can just use the class variable instead of a string!